crucible-install.sh

```diff
@@ -0,0 +1,171 @@
+    fi
+
+    if [ -z "$CRUCIBLE_NAME" ]; then
+        echo "Please enter your full name:"
+        read CRUCIBLE_NAME
+        echo "CRUCIBLE_NAME=\"$CRUCIBLE_NAME\"" >>"$identity"
+    fi
+    if [ -z "$CRUCIBLE_EMAIL" ]; then
+        echo "Please enter your email address:"
+        read CRUCIBLE_EMAIL
+        echo "CRUCIBLE_EMAIL=\"$CRUCIBLE_EMAIL\"" >>"$identity"
+    fi
+}
+
+
+function has_dependency {
+    has_dep=0
+    echo "Checking for $1"
+    if $1 --version >/dev/null 2>&1; then
+        has_dep=1
+    else
+    echo "Attempting to get it"
+    if yum install -y $1 >/dev/null; then
+        has_dep=1
+    else
+        exit_error "You need to install $1 before you can install crucible" $EC_FAIL_DEPENDENCY
+    fi
+fi
+echo "$1: Got it"
+echo
+}
+
+longopts="name:,email:,help"
+opts=$(getopt -q -o "" --longoptions "$longopts" -n "$0" -- "$@");
+if [ $? -ne 0 ]; then
+    exit_error "Unrecognized option specified: $@"
+fi
+eval set -- "$opts";
+while true; do
+    case "$1" in
+       --registry)
+            shift;
+            CRUCIBLE_REGISTRY="$1"
+            shift;
+            ;;
+       --name)
+            shift;
+            CRUCIBLE_NAME="$1"
+            shift;
+            ;;
+        --email)
+            shift;
+            CRUCIBLE_EMAIL="$1"
+            shift;
+            ;;
+        --auth-file)
+            shift;
+            CRUCIBLE_AUTH_FILE="$1"
+            shift;
+            ;;
+        --help)
+            shift;
+            usage
+            exit
+            ;;
+        --)
+            shift;
+            break;
+           ;;
+        *)
+           exit_error "Unexpected argument [$1]"
+           shift;
+           break;
+           ;;
+    esac
+done
+
+if [ "`id -u`" != "0" ]; then
+    exit_error "You must run this as root, exiting" $EC_FAIL_USER
+fi
+
+if [ -z ${CRUCIBLE_REGISTRY+x} ]; then
+    exit_error "You must specify a registry with the --registry option." $EC_FAIL_REGISTRY_UNSET
+fi
+
+identity
+
+for dep in $DEPENDENCIES; do
+    has_dependency $dep
+done
+
+if [ -d $INSTALL_PATH ]; then
+    old_install_path="/opt/crucible-moved-on-`date +%d-%m-%Y_%H:%M:%S`"
+    echo "An existing installation of crucible exists and will be moved to $old_install_path"
+    /bin/mv "$INSTALL_PATH" "$old_install_path"
+fi
+
+echo "Installing crucible in $INSTALL_PATH"
+git clone $GIT_REPO $INSTALL_PATH > $GIT_INSTALL_LOG 2>&1 ||
+    exit_error "Failed to git clone $GIT_REPO, check $GIT_INSTALL_LOG for details" $EC_FAIL_CLONE
+$INSTALL_PATH/bin/install >>"$GIT_INSTALL_LOG" 2>&1 ||
+    exit_error "Failed to execute crucbile-project install, check $GIT_INSTALL_LOG for details" $EC_FAIL_INSTALL
+
+if [ ! -f $CRUCIBLE_AUTH_FILE ]; then
+    exit_error "Crucible authentication file not found. See --auth-file for details." $EC_FAIL_AUTH
+fi
+
+# native crucible install script already created this, only append
+cat << _SYSCFG_ >> $SYSCONFIG
+CRUCIBLE_USE_CONTAINERS=1
+CRUCIBLE_USE_LOGGER=0
```

Comment:
I would like to move to the loger being enabled by default.

Reply:
ack

Reply:
You are 100% certain logger if working fine?  If so, then I guess I am OK with that.

Reply:
I guarantee that it works ;-)